config/locales/spotlight.en.yml

```diff
@@ -1,4 +1,9 @@
 en:
+  time:
+    formats:
+      date:
+        formats:
+          default: '%Y-%m-%d %l:%M%P'
```

Comment:
actually, this part is only applied to the "Date" column (the job's start time).  the elapsed time is handled separately via a rails helper method that isn't influenced by this (AFAIK).